pkg/apis/core/v1alpha1/helper/dataobjects.go

```diff
@@ -22,6 +22,12 @@ const SourceDelimiter = "/"
 
 const NonContextifiedPrefix = "#"
 
+// Prefix combined with installation name is used as label value. Do not change length.
+const InstallationPrefix = "Inst."
+
+// Prefix combined with execution name is used as label value. Do not change length.
+const ExecutionPrefix = "Exec."
```

Comment:
```suggestion
// InstallationPrefix is the prefix combined with installation name is used as label value. Do not change length.
const InstallationPrefix = "Inst."

// ExecutionPrefix is the prefix combined with execution name is used as label value. Do not change length.
const ExecutionPrefix = "Exec."
```

see the godoc style guide https://blog.golang.org/godoc